bigtable-hbase-2.x-parent/bigtable-hbase-2.x/pom.xml

```diff
@@ -84,6 +84,10 @@ limitations under the License.
       <groupId>io.grpc</groupId>
       <artifactId>grpc-stub</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-api</artifactId>
+    </dependency>
```

Comment:
Please separate changes that might change the behavior of the client from ones that dont. ie the clirr plugin doesn't affect endusers, while adding new dependencies will.

Reply:
sg, will fix deps first